language_specific/java/running_examples.rst

```diff
@@ -1,9 +1,22 @@
-Since third-party Java libraries generally don't have a standard place to install them, you will likely have to set the ``CLASSPATH`` variable appropriately to indicate that you wish to use the Xapian Java bindings.
+Since third-party Java libraries generally don't have a standard place
+to install them, you will likely have to set the ``CLASSPATH``
```

Comment:
The phrasing here seems odd: "SInce third-party Java libraries generally don't have a standard place to install them"

Perhaps "Since there isn't a standard place to install third-party Java libraries"?

Or were you trying to say something slightly different to that?